sites/gent/ch_hpc_policies.tex

```diff
@@ -1,4 +1,86 @@
 \chapter{HPC Policies}
 \label{ch:hpc-policies}
 
-Stub chapter
+Everyone can get access and use the HPC-UGent supercomputing infrastructure and services.
+The conditions that apply depend on your affiliation.
+
+\section{Access for staff & academics}
```

Comment:
@epauwels `&` -> `and`
https://github.com/hpcugent/vsc_user_docs/actions/runs/3096533522/jobs/5012197512#step:7:7355

Reply:
Or use `\&`

Reply:
Okay fixed